app/models/claim.rb

```diff
@@ -126,6 +126,10 @@ def submittable?
     valid?(:submit) && !submitted?
   end
 
+  def payroll_gender_missing?
+    !%w[male female].include?(payroll_gender)
```

Comment:
Might be clearer to use [`exclude?`](https://apidock.com/rails/Enumerable/exclude%3F) here